web/frontend/package.json

```diff
@@ -13,44 +13,51 @@
       "import": "./dist/rc.js"
     }
   },
+  "peerDependencies": {
+    "@improbable-eng/grpc-web": "0.15.0",
+    "@viamrobotics/rpc": "0.1.36",
+    "@viamrobotics/sdk": "0.0.56",
+    "google-protobuf": "3.21.2",
+    "three": "0.152.2",
+    "trzy": "0.0.44"
```

Comment:
In my experience, `peerDependencies` and `dependencies` are mutually exclusive categories. Why do we want these packages listed in both?

If they truly are "peer" dependencies, locking them like this is an overly restrictive versioning strategy that risks causing every peer dependency bump that doesn't simply widen the range to become a "breaking" change

Reply:
Oh yep! Agreed. Forgot to remove them from dependencies. Will do.

Concerning locking: I think that's fair, I'll unlock 'em